### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by Namahsivaya7**
Screenshot (78)

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -48,18 +49,21 @@ export default function RootLayout({
...
+          {/* <Header style={{ ...headerStyle, background: colorBgContainer }}>
             <Flex justify="space-between" align="center">
               <Typography.Title level={4}>Meesho</Typography.Title>
               <Link href="/cart">Cart</Link>
             </Flex>
-          </Header>
+          </Header> */}
           <Layout>
-            <Sider
+            {/* <Sider
```
Comment (NagaDurga16) [inline src/app/layout.tsx]: please remove unwanted code

**Review by NagaDurga16 — CHANGES_REQUESTED** [review]

**Comment by Namahsivaya7** [comment]
Screenshot (81)

**Comment by Namahsivaya7** [comment]
Screenshot (79)
Screenshot (80)
Screenshot (81)

**Comment by Namahsivaya7** [comment]
Screenshot (82)
Screenshot (83)

**Review by NagaDurga16 — DISMISSED** [review]

**Review by tulasikurete — APPROVED** [review]

**Comment by Namahsivaya7** [comment]
Screenshot (84)

**Review by NagaDurga16 — APPROVED** [review]

**Comment by Namahsivaya7** [comment]
Screenshot (86)
Screenshot (87)

**Review by arajajyothibabu — APPROVED** [review]

**Pull request merged**